src/charactersheet/services/common/account/messaging/node_service.js

```diff
@@ -102,6 +103,7 @@ function _NodeService(config) {
         self._addXMPPHandlers();
 
         Notifications.party.roster.changed.add(self._getCards);
+        Notifications.party.roster.changed.add(self._subscribeToAll);
```

Comment:
Since both of these methods fire on the same signal? Should they be combined as to reduce the amount of subscriptions?